### DISCUSSION TITLE
Regridding a variable on the ne120_t12 grid to a rectilinear lon/lat grid

### DISCUSSION THREAD

**Discussion by tlsw231** [UXARRAY/uxarray#67] [Q&A]
I have some variables from the CAM model on the unstructured `ne120_t12` grid and would like to regrid them to a `0.25 deg x 0.25 deg` rectilinear `lon/lat` grid (just want to visualize them). This is the first time I am working with CAM output. How does one go about doing this? 

[Edit: I have used the `regrid` function from `xesmf` before for moving between two rectilinear grids but not from unstructured -> rectilinear]

**Comment by JdeJong96** [comment]
One way to regrid the unstructured data is by using the `remap` function from `cdo`. For example, you can run 

`cdo remapcon,n360 infile outfile` 

to perform a first order conservative regridding to a regular 0.25 deg Gaussian grid. There are other interpolation methods possible, described in the [docs](https://code.mpimet.mpg.de/projects/cdo/wiki/Cdo#Documentation) (not all work for this type of grid). Another neat feature is that you can also store the computed grid weights separately, allowing for fast regridding of subsequent files sharing the same grid. For conservative regridding this would look like:

`cdo gencon,n360 infile weightfile`
`cdo remap,n360,weightfile infile outfile`
